core-designsystem/src/main/java/com/sopt/now/component/SoptDialogFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.sopt.now.component
+
+import android.content.DialogInterface
+import android.os.Bundle
+import android.view.View
+import androidx.core.content.ContextCompat
+import com.sopt.now.coreui.base.BindingDialogFragment
+import com.sopt.now.coreui.util.fragment.stringOf
+import com.sopt.now.type.DialogType
+import org.sopt.now.designsystem.databinding.DialogSoptBinding
+
+class SoptDialogFragment(
+    private val dialogType: DialogType = DialogType.LOGOUT,
+    private val clickLeftBtn: () -> Unit = {},
+    private val clickRightBtn: () -> Unit = {},
+    private val onDialogClosed: () -> Unit = {}
```

Comment:
```suggestion
    private val clickLeftBtn: () -> Unit,
    private val clickRightBtn: () -> Unit,
    private val onDialogClosed: () -> Unit
```

{ }는 반환값이 없으니까 지워도 되지 않나요?

Reply:
다이얼로그 만들 때 재정의 할 필요가 없을 경우를 대비해서 넣어두었서요!